File: generated-sources/scala-gatling/mojang-sessions/src/gatling/scala/com/github/asyncmc/mojang/sessions/scala/gatling/model/PlayerTextureURL.scala

package com.github.asyncmc.mojang.sessions.scala.gatling.model


case class PlayerTextureURL (
    /* The URL to the texture, must be in Mojang's domains. */
    _url: String
)
object PlayerTextureURL {
    def toStringBody(var_url: Object) =
        s"""
        | {
        | "url":$var_url
        | }
        """.stripMargin
}

File: generated-sources/kotlin-spring/mojang-sessions/src/main/kotlin/com/github/asyncmc/mojang/sessions/kotlin/spring/model/PlayerTextureURL.kt
package com.github.asyncmc.mojang.sessions.kotlin.spring.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * Wraps the URL to the texture
 * @param url The URL to the texture, must be in Mojang's domains.
 */
data class PlayerTextureURL (

        @get:NotNull 
        @JsonProperty("url") val url: String
) {

}

